airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+import time
+
+
+class AwsGlueJobHook(AwsBaseHook):
```

Comment:
Can we unify the naming convention here? You've used both "AWS" and "Aws" here:

https://github.com/apache/airflow/blob/8b5d9a24167409baf8761ed03cc17e5b7e1dd344/airflow/providers/amazon/aws/hooks/glue.py#L25
https://github.com/apache/airflow/blob/8b5d9a24167409baf8761ed03cc17e5b7e1dd344/airflow/providers/amazon/aws/operators/glue.py#L28
https://github.com/apache/airflow/blob/8b5d9a24167409baf8761ed03cc17e5b7e1dd344/airflow/providers/amazon/aws/sensors/glue.py#L25